scripts/lib/CIME/build.py

```diff
@@ -112,9 +112,206 @@ def _build_model(build_threaded, exeroot, clm_config_opts, incroot, complist,
     return logs
 
 ###############################################################################
-def _build_checks(case, build_threaded, comp_interface, use_esmf_lib,
-                  debug, compiler, mpilib, complist, ninst_build, smp_value,
-                  model_only):
+def post_build(case, logs):
+###############################################################################
```

Comment:
@agsalin , the only recent change (last two months) to this file on the ACME side was one line:

```
@@ -292,6 +292,7 @@ def case_build(caseroot, case, sharedlib_only=False, model_only=False):
                                cimeroot, libroot, lid, compiler)
 
     if not sharedlib_only:
+        os.environ["INSTALL_SHAREDPATH"] = os.path.join(exeroot, sharedpath) # for MPAS makefile generators
```